src/wp-includes/kses.php

```diff
@@ -1468,6 +1470,41 @@ function wp_kses_hair( $attr, $allowed_protocols ) {
 	return $attrarr;
 }
 
+/**
+ * Santizes uris in attributes
+ *
+ * This condenses code that was spread around two functions and several cases. It places the list of attributes
+ * which can have uris in them and checks for ones that can have multiple uri candidates (like srcset)
+ * It ultimately passes everything to wp_kses_bad_protocol() to do the actual work.
+ *
+ * @since ?
+ *
+ * @param string $attrname Attribute name to test against.
+ * @param string $attrvalue Content to filter bad protocols from.
+ * @param string[] $allowed_protocols Array of allowed URL protocols.
+ * @return string Filtered content.
+ */
+function wp_kses_sanitize_uris( $attrname, $attrvalue, $allowed_protocols ) {
+	$uris           = wp_kses_uri_attributes();
+	$uri_candidates = array( 'srcset' );
```

Comment:
Hmm, wondering if this should be an optional forth param? I.e. 
`function wp_kses_sanitize_uris( $attrname, $attrvalue, $allowed_protocols, $uri_candidates = array( 'srcset' ) )`.

Or maybe abstract the code a bit more, and add another function? Seems KSES filtering of arrays of URIs may be good to have for other (eventual) cases?

Also maybe consider a bit different/a bit more descriptive var name for `$uri_candidates`? Perhaps something like `$multi_uri` or just `$uris`?